frontend/src/views/HomeView.vue

```diff
@@ -29,7 +29,14 @@ const { getConfig } = storeToRefs(useConfigStore());
       <p class="text-xl text-center">
         This website uses the <a href="https://bcgov.github.io/common-service-showcase/services/coms.html">Common Object Management Service</a> so
         <a href="https://www2.gov.bc.ca/gov/content/governments/services-for-government/information-management-technology/id-services/idir">IDIR</a> and <a href="https://www.bceid.ca/">BCeID</a>
-        users can <br /> upload and share files.
+        users can <br /> upload and share files.<br /><br />
+        You will need access to a bucket – a file storage container – before you can use BCBox.
+        If you are new to buckets and don't know a bucket manager, contact
+        <a href="https://apps.nrs.gov.bc.ca/int/jira/servicedesk/customer/portal/1/create/701">NRIDS Optimization</a> (NRM only)
+        or your ministry's service desk to get credentials for a bucket. Then come back, log in, and add it in BCBox.
+        <br /><br />
+        If you already have a bucket from outside of BCBox, you can't yet work on existing files
+        from that bucket in BCBox, but we are currently working to make this possible through synchronization.
```

Comment:
Should revisit the `<br />` as it looks a bit visually jarring to have the text forcibly word wrapped like that; it does not flow well when the viewport size changes.

Reply:
Are you referring to the one in the middle of the first sentence; i.e. `users can <br /> upload and share files`? That one was already there previously, but yeah it does look a bit out of place.